src/test_core.py

```diff
@@ -88,7 +88,7 @@ def test_theme_overrides_dirs(self):
     def test_template_renders__multiline_value_as_valid_json(self):
         self.techdocscore.on_config(self.mkdocs_yaml_config)
         env = Environment(
-            loader=PackageLoader("test", self.techdocscore.tmp_dir_techdocs_theme.name),
+            loader=PackageLoader("src", self.techdocscore.tmp_dir_techdocs_theme.name),
```

Comment:
Why do we need this change? I don't see that being done in https://github.com/backstage/mkdocs-techdocs-core/pull/223.

@deejay1 any thoughts on this?

Reply:
seems like it doesn't really matter in this test, as it passes either way

Reply:
When I ran locally the test failed without this change at the time 🤷‍♂️